fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java

```diff
@@ -357,6 +375,21 @@ protected void createDatastream(final String pid, final String dsid, final Strin
         assertEquals(CREATED.getStatusCode(), getStatus(putDSMethod(pid, dsid, content)));
     }
 
+    protected Thread createSNSDatastreamThread(final String pid, final String dsid, final File content)
+            throws IOException {
+        return (new Thread() {
+            public void run() {
+
+            try {
+                logger.info("Attempting to create datastream for object: {} at datastream ID: {}", pid, dsid);
+                getStatus(putDSFileMethod(pid, dsid, content));
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
`printStackTrace()` is bad form. Use a proper `Logger`, if logging a stacktrace is what is wanted.